.github/workflows/finalize-release.yaml

```diff
@@ -16,9 +16,9 @@ jobs:
     name: Cherry-pick Changelogs
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
-          ref: ${{ github.ref }}
+          ref: ${{ github.ref }} # checkouts the branch that triggered the workflow
```

Comment:
based on the triggers you have (L8-12), doesnt this pipeline get run when a new tag has been pushed with the `release/` prefix? how does it know what branch to use from that?

Reply:
Based on the docs I believe its fine:
  | The fully-formed ref of the branch or tag that triggered the workflow run. For workflows triggered by push, this is the branch or tag ref that was pushed.
  
  unless the ref of the tag is different then a ref to the branch? I'm not too familiar with tags, so I'll double check this. Thanks for pointing it out